.github/workflows/subtree.yml

```diff
@@ -70,3 +71,4 @@ jobs:
           path: 'examples/${{ matrix.path }}'
           deploy_key: ${{ secrets.TRPC_DEPLOY_TOKEN }}
           force: true # will force push to the downstream repository
+          branch: main # will push to the main branch of the downstream repository
```

Comment:
Allows us to test subtree on branches without having to clean them up afterwards